.gitattributes

```diff
@@ -1 +1,10 @@
 *.sh eol=lf
+*.md eol=lf
+*.json eol=lf
+*.yaml eol=lf
+.*ignore eol=lf
+*.markdown eol=lf
+nvm-exec eol=lf
+Makefile eol=lf
+bash_completion eol=lf
+
```

Comment:
I can't imagine why we'd have anything else, but we can deal with that as it comes.